FCCSW_ecal/runSlidingWindowAndCaloSim.py

```diff
@@ -316,13 +316,13 @@
 correctCaloClusters = CorrectCaloClusters("correctCaloClusters",
                                           inClusters = createClusters.clusters.Path,
                                           outClusters = "Corrected"+createClusters.clusters.Path,
-                                          numLayers = [12],
+                                          numLayers=[12],
                                           firstLayerIDs = [0],
-                                          lastLayerIDs = [11],
+                                          lastLayerIDs=[11],
                                           readoutNames = [ecalBarrelReadoutNamePhiEta],
-                                          upstreamParameters = [[0.033955208567442975, -3.818122686176795, -146.59497297249345, 0.563447903447204, -3.7906629536351906, -8.569962044554627]],
+                                          upstreamParameters = [[0.03900891447361534, -4.322941016402328, -139.1811369546787, 0.498342628339746, -3.3545078429754813, -13.99996971344221]],
```

Comment:
Are those parameters corresponding to $FCCDETECORS/Detector/DetFCCeeIDEA-LAr/compact/FCCee_DectMaster.xml ?

Reply:
same